### DISCUSSION TITLE
Set up SHVDT on Visual studio 2022

### DISCUSSION THREAD

**Discussion by gyarisuta** [scripthookvdotnet/scripthookvdotnet#1323] [Q&A] [answered]
i cannot get intellisense working, which makes coding harder. if i type vehicle. only generic stuff appears.

**Comment by kagikn** [comment, marked as answer]
Insufficient info to answer with one concrete solution, but you need to reference SHVDN assembly as how ["Manage references in a project" in Microsoft Learn describes](https://learn.microsoft.com/en-us/visualstudio/ide/managing-references-in-a-project?view=vs-2022), either by the ScriptHookVDotNet3.dll file you downloaded or via NuGet. After that, write the statement `using GTA;` at the top of a source code file of your script project, and IntelliSense will be able to show SHVDN classes.

**Reply by gyarisuta** [reply]
i also needed to create a "class .net" project since it wasnt quite working before. but after this and installing the references. it now works! thanks.
